src/SFA.DAS.ApprenticeCommitments.Database/PostDeployment/PostDeployment.sql

```diff
@@ -0,0 +1,27 @@
+﻿/*
+Post-Deployment Script Template							
+--------------------------------------------------------------------------------------
+ This file contains SQL statements that will be appended to the build script.		
+ Use SQLCMD syntax to include a file in the post-deployment script.			
+ Example:      :r .\myfile.sql								
+ Use SQLCMD syntax to reference a variable in the post-deployment script.		
+ Example:      :setvar TableName MyTable							
+               SELECT * FROM [$(TableName)]					
+--------------------------------------------------------------------------------------
+*/
+
+
+/* Developer Role Access */
+
+IF DATABASE_PRINCIPAL_ID('Developer') IS NULL
+BEGIN
+    CREATE ROLE [Developer]
+END

```

Comment:
Would it be more appropriate to call this Role Reporter?

Reply:
Do we need to call this role 'Developer'?

Reply:
Amended to Reporter